iq-scrape-conf.py

```diff
@@ -138,7 +138,9 @@ def org_configuration(org):
     orgconf['component_labels'] = persist_component_labels(org=org['id'])
     orgconf['license_threat_groups'] = persist_license_threat_groups(org=org['id'])
     orgconf['access'] = persist_access(org=org['id'])
+    orgconf['policy'] = persist_policy(org=org['id'])
     orgconf['name'] = org['name']
+    orgconf['policy'] = persist_policy()
```

Comment:
Is this required? It forces the ROOT_ORG policies to be persisted. This is just another Org and so will be done as the script iterates over the array of Orgs.

Reply:
Good catch, I'll remove it.